apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,142 @@
+            >
+                {address}
+            </Link>
+        </div>
+    );
+}
+
+function Amount({ amount, symbol }: { amount: number; symbol?: string }) {
+    return (
+        <div className="flex flex-row items-end gap-1 text-sui-grey-100 ml-6">
+            <Heading as="h4" variant="heading4">
+                {amount}
+            </Heading>
+            {symbol && (
+                <div className="text-sui-grey-80">
+                    <Text variant="bodySmall">{symbol}</Text>
+                </div>
+            )}
+        </div>
+    );
+}
+
+type Recipient = {
+    address: string;
+    amount?: {
+        value: number;
+        unit?: string;
+    };
+};
+
+export interface SenderRecipientProps {
+    sender: string;
+    transferCoin?: boolean;
+    recipients?: Recipient[];
+}
+
+export function SenderRecipient({
+    sender,
+    recipients,
+    transferCoin = true,
+}: SenderRecipientProps) {
+    const multipleRecipients = !!(recipients?.length && recipients?.length > 1);
+    const singleTransferCoin = !!(
+        !multipleRecipients &&
+        transferCoin &&
+        recipients?.length
+    );
+    const primaryRecipient = singleTransferCoin && recipients?.[0];
+    const multipleRecipientsList = primaryRecipient
+        ? recipients?.filter((_, i) => i !== 0)
+        : recipients;
+
+    return (
+        <div className="flex flex-col justify-start h-full text-sui-grey-100 gap-4">
+            <Heading as="h4" variant="heading4" weight="semibold">
+                Sender {singleTransferCoin && '& Recipient'}
+            </Heading>
+            <div className="flex flex-col gap-[15px] justify-center relative">
+                {singleTransferCoin && (
+                    <div className="absolute border-2 border-[#a0b6c3] overflow-y-hidden h-[calc(55%)] w-4 border-r-[transparent] border-t-[transparent] mt-1 ml-1.5 rounded-l border-dotted" />
+                )}
+                <SenderRecipientAddress isSender address={sender} />
+                {primaryRecipient && (
+                    <SenderRecipientAddress
+                        address={primaryRecipient.address}
+                        isCoin
+                    />
+                )}
+                {multipleRecipientsList?.length ? (
+                    <div className="mt-2 flex flex-col gap-2">
+                        <div className=" mt-[5px] mb-2.5">
+                            <Heading
+                                as="h4"
+                                variant="heading4"
+                                weight="semibold"
+                            >
+                                Recipient
+                                {multipleRecipientsList.length > 1 && 's'}
+                            </Heading>
+                        </div>
+                        {multipleRecipientsList?.map((recipient, index) => (
+                            <div
+                                className="flex flex-col gap-2.5 mb-2"
```

Comment:
Why mb-2? Seems like it'll add bottom space we don't want.

Reply:
the spacing between Recipients

Reply:
Seems like gap should take care of that?